clang/test/SemaSYCL/kernel-arg-opt-report.cpp

```diff
@@ -216,12 +216,12 @@ int main() {
 // SPIR-NEXT: String:          'Arg '
 // SPIR-NEXT: Argument:        '7'
 // SPIR-NEXT: String:          ':'
-// SPIR-NEXT: String:          Compiler generated argument for stream,
-// SPIR-NEXT: String:          DecompStream
+// SPIR-NEXT: String:          Compiler generated argument for accessor,
```

Comment:
I haven't looked at it closely but this test and most of the others shouldn't be changing.  This patch should not be modifying kernel arguments generated. If I am understanding this correctly, the only thing that should be changing is how special types are identified. I don't think that should cause any test changes (other than any IR change that adding an attribute to type causes if any)